src/modules/organizations/containers/organizations-container.jsx

```diff
@@ -17,35 +17,67 @@ class OrganizationsContainer extends React.Component {
   }
 
   componentWillUnmount() {
-    this.props.dispatch(setOrganizations([]));
+    this.props.dispatch(setOwnedOrganizations([]));
+    this.props.dispatch(setCollaboratorOrganizations([]));
   }
 
   fetchOrganizations() {
-    apiClient.type('organizations').get({ sort: 'display_name' }).then((orgs) => {
-      this.props.dispatch(setOrganizations(orgs));
-    });
+    apiClient.type('organizations').get({
+      sort: 'display_name',
+      current_user_roles: ['owner'],
+      include: 'avatar',
+    })
+      .then((orgs) => {
+        const orgsWithAvatar = [];
+        orgs.forEach((org) => {
```

Comment:
I don't think a `forEach` here, then pushing and dispatching on lines 36 and 37 is the best approach. I tried implementing something with `.map` and tried nested `.then`'s but couldn't get anything to work.